denguefever_tw/dengue_linebot/utils.py

```diff
@@ -1,18 +1,26 @@
+from django.conf import settings
+
 import logging
+import requests
+from time import sleep
+
+from selenium import webdriver
+from pyvirtualdisplay import Display
 
```

Comment:
There is no need to add this blank line.

Reply:
Fixed